spk/jellyfin/Makefile

```diff
@@ -1,12 +1,12 @@
 # Remember to also update jellyfin-web
 SPK_NAME = jellyfin
-SPK_VERS = 10.8.9
-SPK_REV = 9
+SPK_VERS = 10.8.10
+SPK_REV = 10
 SPK_ICON = src/jellyfin.png
 WIZARDS_DIR = src/wizard/
 DSM_UI_DIR = app
 
-DEPENDS = cross/libstdc++ cross/jellyfin cross/jellyfin-web
+DEPENDS = cross/libstdc++ cross/libicu cross/jellyfin cross/jellyfin-web
```

Comment:
before we add libicu to all dotnet packages we should try to find out, what the reason is, that libicu on DSM 7.2 Beta is not found (libicu version 64.2 is available in /usr/lib/) .
_Probably synology will fix this for the final release of DSM 7.2_

Reply:
@mreid-tt I just fixed the libicu issue by adding /usr/lib to LD_LIBRARY_PATH like this (DSM 7.2 beta on synology_kvmx64_virtualdsm)

```
SERVICE_COMMAND="env LD_LIBRARY_PATH=${SYNOPKG_PKGDEST}/lib:/usr/lib ${SYNOPKG_PKGDEST}/share/jellyfin ${JELLYFIN_ARGS}"
```


without this fix I got the known error:
```
Process terminated. Couldn't find a valid ICU package installed on the system. ...
```

Reply:
Oops, the LD_LIBRARY_PATH fix did not work... (finally it failed with the same error message...)

Reply:
Yeah, it would be useful to identify if this issue is only in the DSM 7.2 beta but for now this was the only way I found to get the app to launch. We can probably look at re-testing this once DSM 7.2 is final. What are your thoughts?

Reply:
Finally, I found that the problem is the other way round...

We must not include stdlibc++ in the jellyfin package for DSM >= 7. Neither we need cross/libicu at all.
For DSM < 7 the version of stdlibc++ is too old, and we must include stdlibc++ in the package.

For DSM >= 7, we must not include cross/stdlibc++ nor cross/libicu and must not add LD_LIBRARY_PATH in service-setup.sh.

We could patch the rpath in the jellyfin executable for packages created for DSM <7 at build time to avoid the need of LD_LIBRARY_PATH at all.
_only packages depending on dotnet that use the internal updater require LD_LIBRARY_PATH as we cannot patch updated executables._
For optional dependency on cross/libstdc++ and patching with `patchelf` see the example in spk/dotnet-runtime/Makefile.

Reply:
BTW:  jellyfin  `10.8.9-9` runs on my DSM 7.2 beta on synology_kvmx64_virtualdsm when removing the LD_LIBRARY_PATH in scripts/service-setup and/or by deletion of /var/packages/jellyfin/target/lib folder.

Reply:
> Yeah, it would be useful to identify if this issue is only in the DSM 7.2 beta but for now this was the only way I found to get the app to launch. We can probably look at re-testing this once DSM 7.2 is final. What are your thoughts?

It was our mistake, synology has done it right.

Reply:
This is great news! I've pushed an update based on what you have stated, let me know if it looks correct.

Reply:
> For optional dependency on cross/libstdc++ and patching with `patchelf` see the example in spk/dotnet-runtime/Makefile.

I didn't use the `OPTIONAL_DEPENDS` variable. It seems to work without it but should I add it?

Reply:
> I didn't use the `OPTIONAL_DEPENDS` variable. It seems to work without it but should I add it?

Yes the build _works_ as `OPTIONAL_DEPENDS` is for the evaluation of package dependencies to build.

I highly recommend to add cross/stdlibc++ with OPTIONAL_DEPENDS. This is required for the github build action to trigger a build of spk/jellyfin when something changed in cross/stdlibc++.
The evaluation of dependencies is not able to find dependencies that are evaluated in the Makefile (see note in spksrc.dependency-tree.mk).

Reply:
Thanks for the clarification. It has been added.